wayfinder/manifest.json

```diff
@@ -0,0 +1,65 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a68bad83-ba55-4350-a913-2f98bb667bad",
+  "app_id": "wayfinder",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Wayfinder metrics to Datadog",
+    "title": "Appvia Wayfinder",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Wayfinder Sample Datadog Dashboard",
+        "image_url": "images/wayfinder-datadog-dash.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Wayfinder UI",
+        "image_url": "images/wayfinder-ui.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Developer Tools",
+      "Category::Kubernetes",
+      "Category::Metrics",
+      "Category::Orchestration",
+      "Supported OS::Linux",
+      "Supported OS::Windows"
```

Comment:
Please also add tags for submitted or queried data types. For example, if your integration submits metrics, you would add:

"Submitted Data Type::Metrics"

Reply:
done